### PR TITLE
added missing publish-uploaded-assets.xml workflow

### LINKED ISSUE #4715: publish-uploaded-assets.xml workflow is missing
publish-uploaded-assets.xml workflow was removed with OC 11.x. This workflow runs automatically when an asset is updated via admin ui and can't be run manually. Backend implementation (automatic initiation of this WF when an event asset is updated) of this WF is still working.

**To Reproduce**

Steps to reproduce the behavior on OC 11.x and later:
1. Go to admin ui
2. Click on any events assets
3. Add a new asset
4. See nothing happening

**Expected behavior**

Asset added, and published automatically for the event.

**Server environment:**
 - Version 12.5

### PR DESCRIPTION
**Pull request by aliyevTU**
Closes https://github.com/opencast/opencast/issues/4715

### CODE REVIEW

**Comment by aliyevTU** [comment]
Submitted ICLA. Will it auto detect my new ICLA submission or do I have to manually trigger it?

**Comment by mliradelc** [comment]
> Submitted ICLA. Will it auto detect my new ICLA submission or do I have to manually trigger it?

It will be automatically detected, thanks for your PR, it will be reviewed next Tuesday on the weekly tech meeting. You can also participate. [Details are on the docs page](docs.opencast.org)

**Review by mliradelc — CHANGES_REQUESTED** [review]
Hi, I'm reviewing your code. Can you rol back all the `package.json`and `package-lock.json` changes?

Those changes are maintained using dependabot and are out of the scope of this PR.

And again, thanks for you PR! 🙂

**Comment by aliyevTU** [comment]
> Hi, I'm reviewing your code. Can you rol back all the `package.json`and `package-lock.json` changes?
> 
> Those changes are maintained using dependabot and are out of the scope of this PR.
> 
> And again, thanks for you PR! 🙂


Update Bot added those changes. I will remove them now.

**Comment by aliyevTU** [comment]
> Hi, I'm reviewing your code. Can you rol back all the `package.json`and `package-lock.json` changes?
> 
> Those changes are maintained using dependabot and are out of the scope of this PR.
> 
> And again, thanks for you PR! 🙂

Done. Thank you 😃

File: etc/workflows/publish-uploaded-assets.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <!-- Clean up work artifacts -->
+
+    <operation
+        id="cleanup"
+        fail-on-error="false"
+        description="Remove temporary processing artifacts">
+      <configurations>
+        <configuration key="delete-external">true</configuration>
+        <!-- FixMe Don't clean up ACLs until workflow service no longer looks for them in the WFR. -->
+        <configuration key="preserve-flavors">security/*</configuration>
```
Comment (mliradelc) [inline etc/workflows/publish-uploaded-assets.xml]: @gregorydlogan @KatrinIhler  What do you think about this line?

I think this should go to `r/12.x`

**Review by mliradelc — COMMENTED** [review]
Seems good for me but there is a line that I have doubts, I just asked some colleagues about it.

File: etc/workflows/publish-uploaded-assets.xml (hunk 1 above)
Comment (aliyevTU) [reply etc/workflows/publish-uploaded-assets.xml]: That line been there for quite some time, so I left it there on my PR.

Comment (KatrinIhler) [reply etc/workflows/publish-uploaded-assets.xml]: As far as I'm aware, that is still necessary,

**Comment by mliradelc** [comment]
Finally, can you change the target branch to r/12.x?

**Comment by aliyevTU** [comment]
> Finally, can you change the target branch to r/12.x?

No, accept it as it is, or reject it.

**Comment by mtneug** [comment]
Closed in favor of #4757 for a rebase to r/12.x. This change is still contributed to you @aliyevTU.

**Comment by mtneug** [comment]
> No, accept it as it is, or reject it.

Thank you, @aliyevTU. We value all contributions and do not want to discourage people. At the same time, we want Opencast adopters running older supported versions to benefit from new features and bug fixes. According to our set guidelines, we thus may ask contributors to rebase a PR to a different target. We hope you will contribute again in the future and if you need any help in "running the right Git commands", feel free to reach out. Thanks again.

**Comment by aliyevTU** [comment]
> > No, accept it as it is, or reject it.
> 
> Thank you, @aliyevTU. We value all contributions and do not want to discourage people. At the same time, we want Opencast adopters running older supported versions to benefit from new features and bug fixes. According to our set guidelines, we thus may ask contributors to rebase a PR to a different target. We hope you will contribute again in the future and if you need any help in "running the right Git commands", feel free to reach out. Thanks again.

Thank you. I tried myself and I literally could not change the branch. So I thought the only possible way to solve it without spending more time on it, is to either accept it directly to develop or to do the whole PR from scratch. Thank you for taking over this part. 

Problems I had was with too many merge conflicts when I changed the branch. My forked OC version only had branches till r/6.x, I tried adding r/12.x from official repo, and then changing the PR to my new forked r/12.x branch. But it generated so much merge conflicts that I just undone that, and informed @mliradelc that I can not change the branch on this PR.

**Pull request closed without merging**